### PR TITLE
test: try to unflake fixtures tests

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

**Comment by pavelfeldman** [comment]
They never fail after my tracing fix. Coincidence?

**Comment by dgozman** [comment]
They still fail for me locally.

File: test/fixtures.spec.js [hunk 1]
```diff
@@ -97,20 +97,14 @@ module.exports.describe = function({testRunner, expect, product, browserType, pl
     if (!WIN) {
```
Comment (aslushnikov) [inline test/fixtures.spec.js]: this should be `describe.skip(WIN)`?

**Review by aslushnikov — APPROVED** [review]

File: test/fixtures.spec.js (hunk 1 above)
Comment (dgozman) [reply test/fixtures.spec.js]: Done.

**Pull request merged**